process/consensus.md

```diff
@@ -9,24 +9,96 @@ Consensus is critical in a standards process:
   over time, break down. Recording dissent is therefore a critical part of
   building consensus.
 
-## In-person meeting consensus
+## Informal consensus
 
-For in-person meetings, champions are expected to list points for which they
-will seek consensus in the meeting agenda, and new consensus points can be added
-in-person as the discussion proceeds.
+It is critical that work progresses between meetings: agreed-upon designs need
+to move forward, and new ideas need to reach some level of maturity before being
+discussed before the full CG. To that end, the group can reach consensus
+informally via discussion either on GitHub repositories under the WebAssembly
+organization or in meetings.
 
-Polling consensus is done by the chair:
+We introduce the following concepts to help determine when informal consensus is
+sufficient:
+
+* Contributors of *different affiliation* are individuals who are affiliated
+  with different companies or institutions.
+* *Interested parties* are individuals, companies, or institutions who usually
+  participate in the WebAssembly Community Group or Working Group and are
+  interested in a particular topic.
+* *Small group* is a subset of Community Group and Working Group participants
+  who decide to collaborate on a single targeted proposal.
+
+We differentiate the following cases:
+
+1. Editors are empowered to make changes that have no technical impact without
+   explicit group decision. GitHub issues and pull requests are sufficient.
+   Their title should identify them as editorial. Closing or merging them after
+   a full work day of positive review by at least one contributor of different
+   affiliation is acceptable as long as there is no objection. Editors shall
+   revisit the editorial nature of the change if there is any contention.
+
+2. Small technical changes or additions can be discussed in GitHub issues and
+   pull requests. It is the author's responsibility to ensure that interested
+   parties sign off. Simply mentioning people on GitHub may go unnoticed, the
+   onus remains on the author to contact interested parties and obtain their
+   feedback. Closing or merging after a full week of positive review by at least
+   three contributors of different affiliations is acceptable as long as there
+   is no objection. Consensus will be deemed to not have been reached if
+   interested parties did not sign off. At any point in time a contributor can
+   request that final consensus be delayed to a CG meeting. In this case, the
+   chair puts the item on the group's agenda of upcoming discussions.
+
+3. Substantial technical changes or additions are developed in their own GitHub
```

Comment:
maybe it's worth making it explicit that this refers to the usual proposal phase process (with a link to phases.md)?